Library/Homebrew/utils/fork.rb

```diff
@@ -51,7 +51,10 @@ def self.safe_fork
           # to rescue them further down.
           if e.is_a?(ErrorDuringExecution)
             error_hash["cmd"] = e.cmd
-            error_hash["status"] = e.status.exitstatus
+            error_hash["status"] = {
+              exitstatus: e.status.exitstatus,
```

Comment:
Can `.status` ever be `nil`?

Reply:
I don't think so. Technically from an API standpoint it could be an integer (or now, a hash) though - it just never has come up as the only thing calling `ErrorDuringExecution` with an integer or hash is the `safe_fork` code itself.

Reply:
I think being extra safe on this would make sense and not making any assumptions about the presence or type of `status`

Reply:
Yeah I've added a `Process::Status` type check now.